web_app.py

```
@@ -1856,24 +1893,35 @@ def _chat_core(
                         self._trail_mindreading_after_reply,
                         (minister_name, answer_text, chat_turn_id),
                         "audience-p5-mindreading",
+                        ticket_key=("turn", int(chat_turn_id)),
                     )
-                    # #501：叙事抽取落账与读心并行尾随（各自 pending ownership，P5）。
+                    # #501：叙事抽取落账与读心并行尾随（各自队列票据，P5）。
                     self._spawn_extraction_trail(minister_name, answer_text, chat_turn_id)
+                    if not gate_already_held:
+                        self._complete_pending_write(pending_ticket)
+                        pending_ticket = None
                 # #544：非流式折进等待窗——判官以超时封顶后与回话同到；超时则空清单先行。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)